core/src/shred_fetch_stage.rs

```diff
@@ -111,17 +134,23 @@ impl ShredFetchStage {
                 )
             })
             .collect();
-
         let modifier_hdl = Builder::new()
             .name("solana-tvu-fetch-stage-packet-modifier".to_string())
             .spawn(move || {
+                let repair_context = repair_context.as_ref().map(|(socket, cluster_info)| {
+                    (
+                        <&UdpSocket>::from(socket),
+                        <&ClusterInfo>::from(cluster_info),
```

Comment:
wouldn't just `(&socket, &cluster_info)` compile here?

Reply:
unfortunately no, but changed to use `.as_ref()`